clusterloader2/testing/load/config.yaml

```diff
@@ -0,0 +1,125 @@
+name: Load
+automanagedNamespaces: 3
+tuningSets:
+- name: Uniform100qps
+  qpsLoad:
+    qps: 100
+- name: Uniform1.4qps
+  qpsLoad:
+    qps: 1.4
+steps:
+- measurements:
+  - Identifier: GlobalTimer
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: CreatePodsTimer
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: APICallLatency
+    Method: APICallLatency
+    Params:
+      action: reset
+# Create RCs
+- phases:
+  - namespaceRange:
+      min: 1
+      max: 3
+    replicasPerNamespace: 1
+    tuningSet: Uniform100qps
+    objectBundle:
+    - basename: service
+      objectTemplatePath: service.yaml
+      templateFillMap:
+        RCBasename: rc
+    - basename: rc
+      objectTemplatePath: rc.yaml
+      templateFillMap:
+        ReplicasMin: 30
+        ReplicasMax: 30
+- measurements:
+  - Identifier: WaitForRunningRCs
+    Method: WaitForControlledPodsRunning
+    Params:
+      kind: ReplicationController
+      labelSelector: kind = loadRC
+      timeout: 2m
+- measurements:
+  - Identifier: CreatePodsTimer
+    Method: Timer
+    Params:
+      action: stop
+      label: Pods creation
+  - Identifier: ScaleRCsTimer
+    Method: Timer
+    Params:
+      action: start
+# Scale RCs
+- phases:
+  - namespaceRange:
+      min: 1
+      max: 3
+    replicasPerNamespace: 1
+    tuningSet: Uniform1.4qps
+    objectBundle:
+    - basename: service
+      objectTemplatePath: service.yaml
+      templateFillMap:
+        RCBasename: rc
+    - basename: rc
+      objectTemplatePath: rc.yaml
+      templateFillMap:
+        ReplicasMin: 15
+        ReplicasMax: 45
+- measurements:
+  - Identifier: WaitForScaledRCsRunning
+    Method: WaitForControlledPodsRunning
+    Params:
+      kind: ReplicationController
+      labelSelector: kind = loadRC
+      timeout: 2m
+- measurements:
+  - Identifier: ScaleRCsTimer
+    Method: Timer
+    Params:
+      action: stop
+      label: Scaling RCs
+  - Identifier: DeleteAllPodsTimer
+    Method: Timer
+    Params:
+      action: start
+# Delete RCs
+- phases:
+  - namespaceRange:
+      min: 1
+      max: 3
+    replicasPerNamespace: 0
+    tuningSet: Uniform100qps
+    objectBundle:
+    - basename: service
+      objectTemplatePath: service.yaml
+      templateFillMap:
+        RCBasename: rc
+    - basename: rc
+      objectTemplatePath: rc.yaml
+      templateFillMap:
+        ReplicasMin: 30
```

Comment:
Do you really need to provide this replicas info for RC when deleting them? For deleting, don't you just need the name and namespace of the object?

Reply:
It's true that I only need name and namespace, but...
When template is read, all template functions are evaluated. If there is no parameter provided, the function will panic. That's why parameters are always required.

Reply:
Right. But in principle deletions are sth that shouldn't need any more information except probably the template.

Reply:
OK. I've change the code. It's not required to provide mapping now (for delete operation).

Reply:
I've also removed mapping from deletion parts of test configs.

Reply:
Thanks.